addons/mail/static/src/new/composer/composer.js

```diff
@@ -60,7 +60,11 @@ export class Composer extends Component {
     setup() {
         this.messaging = useMessaging();
         this.store = useStore();
+        useSubEnv({
+            inComposer: true,
+        });
```

Comment:
Only composer `useAttachmentUploader` needs `is_pending` being set.
I reverted most changes and just passed `isPending=true` here.